.ci/hardware.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -e -v
+
+NAME=$0
+COMMAND=$1
+
+if [[ "$COMMAND" == "install" ]]; then
+    pip install coverage  # needed for the uploading in after_success
+    openssl aes-256-cbc -K $encrypted_e0365add3e93_key -iv $encrypted_e0365add3e93_iv -in .ci/travis_rsa.enc -out ~/.ssh/id_rsa -d
+    chmod 600 ~/.ssh/id_rsa
+    echo -e "$INTELHOST_INFO" >> ~/.ssh/config
+    ssh -o StrictHostKeyChecking=no loihihost "echo 'Connected to loihihost'"
```

Comment:
Yeah, and when you're reading the Travis-CI log it's helpful to have a marker for where you are at in the script (since all the above happens silently).